sanic/response.py

```diff
@@ -316,6 +317,7 @@ def html(body, status=200, headers=None):
     if _range:
         headers['Content-Range'] = 'bytes %s-%s/%s' % (
             _range.start, _range.end, _range.total)
+        status = 206
```

Comment:
this may be an overblown worry but what if someone sets a specific status for their request, this will silently change it on them if its a range request. should we set the default to None so we can tell if the caller provided a status?

Reply:
@abuckenheimer Or just check to see if status is already set, and if it isn't, then set the status?

Reply:
That does makes sense, but the default `status` argument value given to the function is `200` and the `_range` keyword needs to be given to the function as well to enter in this condition, which **should** send a `status` value of `206`, so I think that hard-coding it might not be an issue :wink:

Reply:
[update/continue] unless an error occur or something, will the status still be `206` in this case? Just if some kind of read error occurs, which will be prior to the `status` change ... Is there any test in case it seeks for a start point that doesn't exists or try to reads more bytes than it is requested?